e2e-tests/demand-backup/run

```diff
@@ -90,24 +91,23 @@ function main {
 	sleep 30
 
 	gsutil rm -r gs://${BUCKET}/backrestrepo/${cluster}-backrest-shared-repo
-	sleep 30
+
 
 	desc 'Testing backup/restore from local'
 	apply_configs ${cluster}
 	spinup_pgcluster "${cluster}" "${src_dir}/deploy/cr.yaml" "local" "${cluster}-custom-config"
 	prepare_n_restore ${cluster} "posix" "full"
 
 	kubectl_bin delete perconapgcluster/${cluster}
-	sleep 30
+	sleep 60
```

Comment:
I believe we need to do something with 1-2 minute freeze inside operator

Reply:
As I understand operator didn't freeze but pod itself